UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -21,6 +22,10 @@ public sealed class ConfigureJoinableTaskFactoryAttribute : Attribute, ITestActi
 
         public void BeforeTest(ITest test)
         {
+            GitExtensionsFormBase.InstanceRegistry.Clear();
```

Comment:
Should we assert there is nothing in the dictionary?

Reply:
This is asserted in `AfterTest`. Here we must get rid of eventual remainders from the previous failed test.